msteams-platform/messaging-extensions/high-quality-message-extension.md

```diff
@@ -651,6 +651,16 @@ Message extensions respond to a user input with an Adaptive Card. An Adaptive Ca
 * Adaptive Cards must include a URL as part of the [metadata](https://adaptivecards.io/explorer/Metadata.html), which allows cards to be easily copied from one hub to another. [*Recommended*]
 * Apart from thumbnails, any image in an Adaptive Card must have an alt-text. [*Recommended*]
 
+## Security best practices
+
+If your message extension plugin uses URL-based dialogs (task modules), it’s important to note that web browsers are [deprecating third-party cookies](https://blogs.windows.com/msedgedev/2024/03/05/new-privacy-preserving-ads-api/). To ensure your plugin remains functional and secure, request you to implementing the following best practices:
```

Comment:
I think the last clause has a typo: "request you to implementing the following best practices".

Should that be something like "please implement the following best practices"?

Reply:
Thanks! Updated with suggested.